BioCrypt-Personal-V0.4 Source-code/main.py

```
@@ -0,0 +1,1814 @@
+    file_listbox = Listbox(
+        vault_window,
+        font=("Courier", 12),
+        width=90,
+        height=20,
+        fg=var1["bg"],
+        bg=var2["fg"]
+    )
+    file_listbox.pack(pady=10)
+
+    # Replace add_file function with simple file dialog
+    def add_file():
```

Ruff (0.12.2)

Redefinition of unused `add_file` from line 957

(F811)